src/main/java/com/powsybl/nad/model/Edge.java

```diff
@@ -6,16 +6,9 @@
  */
 package com.powsybl.nad.model;
 
-import java.util.Optional;
-
 /**
  * @author Florian Dupuy <florian.dupuy at rte-france.com>
  */
-public interface Edge {
-
-    String getDiagramId();
-
-    String getEquipmentId();
+public interface Edge extends Identifiable {
```

Comment:
An empty interface! This is bad :sweat: (called marker interface)
We should either delete it or reconsider the edge hierarchy. But let's do this in another PR. We should create the corresponding issue once this PR is merged, could you keep this in mind?